homeassistant/components/medcom_ble/sensor.py

```diff
@@ -0,0 +1,113 @@
+"""Support for Medcom BLE radiation monitor sensors."""
+from __future__ import annotations
+
+import logging
+
+from medcom_ble import MedcomBleDevice
+
+from homeassistant import config_entries
+from homeassistant.components.sensor import (
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import CONNECTION_BLUETOOTH, DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import StateType
+from homeassistant.helpers.update_coordinator import (
+    CoordinatorEntity,
+    DataUpdateCoordinator,
+)
+
+from .const import DOMAIN, UNIT_CPM
+
+_LOGGER = logging.getLogger(__name__)
+
+SENSORS_MAPPING_TEMPLATE: dict[str, SensorEntityDescription] = {
+    "cpm": SensorEntityDescription(
+        key="cpm",
+        translation_key="cpm",
+        native_unit_of_measurement=UNIT_CPM,
+        state_class=SensorStateClass.MEASUREMENT,
+        icon="mdi:radioactive",
+    ),
+}
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: config_entries.ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up Medcom BLE radiation monitor sensors."""
+
+    coordinator: DataUpdateCoordinator[MedcomBleDevice] = hass.data[DOMAIN][
+        entry.entry_id
+    ]
+
+    # See the Airthings BLE integration: this template system
+    # lets us change some units if we want to. We could use this
+    # in the future to map CPMs to dose rate, but this depends
+    # on the device calibration - the device only tells us the CPM.
+    # Another approach would be to store the calibration value in
+    # Home assistant directly as a configuration.
```

Comment:
We generally don't want to reference other unrelated integrations in comments since they can change and than the comments would be wrong